### PR TITLE
Added: color-rgba-fallback PostCSS Plugin

### PR DESCRIPTION
**Pull request by magsout**

### CODE REVIEW

File: test/fixtures/cases/example.expected.css [hunk 1]
```diff
@@ -33,7 +34,7 @@ a {
   -webkit-transition: color 1s;
           transition: color 1s; /* autoprefixed ! */
 }
-a:hover { color: rgba(255, 255, 255, 0.5) }
+a:hover { color: #FFFFFF; color: rgba(255, 255, 255, 0.5) }
```
Comment (MoOx) [inline test/fixtures/cases/example.expected.css]: wat ? your plugin output UPPERCASE HEXA COLOR ? It's a scandal :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# 1.3.3 - 2015-04-15
```
Comment (MoOx) [inline CHANGELOG.md]: 1.3.0. Same in package.json please :)

File: index.js [hunk 3]
```diff
@@ -43,7 +44,8 @@ var features = {
   fontVariant: function(options) { return require("postcss-font-variant")(options)},
   filter: function(options) { return require("pleeease-filters")(options)},
   rem: function(options) { return require("pixrem")(options)},
-  pseudoElements: function(options) { return require("postcss-pseudoelements")(options)},
+  pseudoElements: function(options) {return require("postcss-pseudoelements")(options)},
```
Comment (MoOx) [inline index.js]: please readd the space between `{return`. Same for the line you added below.

File: package.json [hunk 4]
```diff
@@ -44,6 +44,7 @@
     "postcss-color-hex-alpha": "^1.1.0",
     "postcss-color-hwb": "^1.1.0",
     "postcss-color-rebeccapurple": "^1.1.0",
+    "postcss-color-rgba-fallback": "^0.1.0",
```
Comment (MoOx) [inline package.json]: I would prefer 1.0.0. Semver works better with that :)

**Comment by MoOx** [comment]
Except the small details, it looks good :)

File: test/fixtures/cases/example.expected.css (hunk 1 above)
Comment (magsout) [inline test/fixtures/cases/example.expected.css]: it's `color-string`

Comment (MoOx) [inline test/fixtures/cases/example.expected.css]: ok. I will handle that some day so :)

File: package.json (hunk 4 above)
Comment (MoOx) [inline package.json]: Can't you push 1.0.0 for this ?

**Comment by magsout** [comment]
Done

**Comment by MoOx** [comment]
looks good. publishing :)

**Comment by MoOx** [comment]
1.3.0 published ! Thanks !

**Comment by bloodyowl** [comment]
does it ignores the declaration when you declared the fallback yourself?

**Comment by MoOx** [comment]
That might be handled, I were thinking about that. I don't think it is for now.

**Comment by magsout** [comment]
@bloodyowl 

>does it ignore the declaration when you declared the fallback yourself?

no

@MoOx 

>That might be handled, I were thinking about that. I don't think it is for now.

You think it's relevant ? If you manage yourself the fallback, disabled the feature ?

**Comment by MoOx** [comment]
I thin before introducing a fallback, you should just look if there is not one before already (eg: same property name with an hexa) (on runtime, for each change your are going to make)

**Comment by magsout** [comment]
>does it ignore the declaration when you declared the fallback yourself?


@bloodyowl it's ok now.

**Comment by MoOx** [comment]
:+1:  * 2

**Pull request merged**